tests/Sinch.Tests/Core/HttpTests.cs

```diff
@@ -76,6 +76,48 @@ public async Task ForceNewTokenOnlyOnce()
             _httpMessageHandlerMock.VerifyNoOutstandingExpectation();
         }
 
+        // [Fact]
```

Comment:
To update in next PR:
* description to clarify test intentions 
* name
* use `expired` header here as well